QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,115 @@
+This document is a declaration of software quality for the `libconsole-bridge-dev` ROS external dependency, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `libconsole-bridge-dev` Quality Declaration
+
+The ROS external dependency `libconsole-bridge-dev` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`libconsole-bridge-dev` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`libconsole-bridge-dev` is not yet at a stable version, i.e. `>= 1.0.0`.
+
+### Public API Declaration [1.iii]
+
+All symbols in the installed headers are considered part of the public API.
+
+### API Stability Policy [1.iv]
+
+`libconsole-bridge-dev` is used as an upstream package within the ROS2 ecosystem, and is pinned to a particular version to guarantee it will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
```

Comment:
This package hasn't changed much in the last few years. I would say instead that the API has been stable for several years, but if a breaking change is introduced, ROS 2 distributions may need to pin to a specific major version.